src/components/sidebar.tsx

```diff
@@ -43,6 +43,7 @@ export default () => {
           <SidebarLink to="/inclusion/">Inclusive Language</SidebarLink>
           <SidebarLink to="/translations/">Translations</SidebarLink>
           <SidebarLink to="/code-review/">Code Review</SidebarLink>
+          <SidebarLink to="/processing-tickets/">Processing Issues & PRs</SidebarLink>
```

Comment:
Might be good to add a reference from https://github.com/getsentry/develop/blob/master/src/docs/index.mdx too.

Reply:
Deciding against this since the docs are geared towards active developers, remember this is just a placeholder until we get open.sentry.io going.